Pipfile

```diff
@@ -7,6 +7,7 @@ verify_ssl = true
 
 [packages]
 quart = "*"
+hashids = "*"
 
 [requires]
-python_version = "3.7"
+#python_version = "3.7"
```

Comment:
Why is this commented, according to @jonay2000 it doesn't work on 3.8 for example

Reply:
nvm, it was fake news